public/javascript/pump/view.js

```diff
@@ -1588,13 +1588,38 @@
         tagName: "div",
         className: "row comment-form",
         events: {
-            "submit .post-comment": "saveComment"
+            "submit .post-comment": "saveComment",
+            "click .close-btn": "cancelComment"
         },
         ready: function() {
             var view = this;
             view.$('textarea[name="content"]').wysihtml5({
                 customTemplates: Pump.wysihtml5Tmpl
             });
+
+            // Close other comments instances if are empty
+            $(".replies .comment-form").each(function(index, el) {
+                var $el = $(el),
+                    html = $el.find('textarea[name="content"]').val();
+
+                if (!html || html.length === 0) {
+                    $el.remove();
+                }
+            });
+        },
+        cancelComment: function() {
+            var view = this,
+                html = view.$('textarea[name="content"]').val();
+
+            if (html.length === 0) {
+                view.remove();
+                return;
+            }
+            Pump.areYouSure("Look like have content, are you sure close this comment?", function(err, sure) {
```

Comment:
This should probably be shortened, something like "You sure? You can't get this comment back!"

Reply:
Yeah, better